core/src/main/clojure/xtdb/indexer.clj

```diff
@@ -62,6 +62,13 @@
 
 (def ^:private abort-exn (err/runtime-err :abort-exn))
 
+(def ^:private forbidden-tables #{"xt/txs" "xt/tx_fns"})
```

Comment:
could do with checking the schema for this one - I'd assume `xt.*` (_except_ xt/tx_fns, ironically - I would guess that this should break the tx-fn tests as new tx-fns are just inserted as put-docs), `information_schema.*` and `pg_catalog.*`?